sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/AclDescriptorFactorySpec.scala

```diff
@@ -54,7 +55,7 @@ class AclDescriptorFactorySpec extends AnyWordSpec with Matchers {
     "generate a default ACL file descriptor with deny code" in {
       val extension = lookupExtension[MainDenyWithCode].get
       val denyCode = extension.getAcl.getDenyCode
-      denyCode shouldBe 7
+      denyCode shouldBe Status.Code.ALREADY_EXISTS.value()
```

Comment:
Wrong import? better to use the acl one instead of the grpc one?

Reply:
yes. changed